src/plugins/action_create_user/src/createUser.js

```diff
@@ -1,78 +0,0 @@
-"use strict";
```

Comment:
I didn't understood why this file was removed

Reply:
Hmm, probably because I thought I'd have to make changes to the `createUser` action, and so need to delete the .js file first so it doesn't take priority over the .ts file. Guess I forgot to rebuild. 

Re-added it now. Ignore any changes in it if there are any.